### DISCUSSION TITLE
[BUG REPORT] The new UI has a bug

### DISCUSSION THREAD

**Discussion by kingcc** [community/community#58296] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The new UI has a bug,
I can't get back the drop-down menu in the top right corner.

https://i.ibb.co/3WDyxPH/Desktop-Screenshot-2023-06-17-14-16-47-23-2.png: ![https://i.ibb.co/3WDyxPH/Desktop-Screenshot-2023-06-17-14-16-47-23-2.png](https://i.ibb.co/3WDyxPH/Desktop-Screenshot-2023-06-17-14-16-47-23-2.png)

**Comment by the-officialjosh** [comment]
If you're experiencing a bug with the new GitHub UI where the dropdown menu in the top right corner appears to be stuck and you're unable to access it, there are a few steps you can try to resolve the issue:

1. **Clear your browser cache**: Sometimes, cached data can interfere with the functionality of a website. Clearing your browser cache can help in refreshing the page and resolving any related issues. You can typically find this option in your browser's settings.

2. **Disable browser extensions**: Certain browser extensions or plugins might conflict with the functionality of the GitHub UI. Try disabling any extensions temporarily and see if the dropdown menu starts working properly.

3. **Try a different browser**: If the issue persists, try accessing GitHub using a different browser. This can help determine if the problem is specific to the browser you're currently using.

4. **Report the issue**: If none of the above steps resolves the problem, it's a good idea to report the issue to GitHub's support team or on their community forums. They can provide specific guidance or escalate the issue to their development team for investigation and resolution.

Remember to provide specific details about your browser version and any relevant error messages or behaviour patterns you're experiencing. This information will assist the support team in identifying the cause of the problem more effectively.

**Reply by ghostinhershell** [reply]
@the-officialjosh While we encourage non-AI answers to questions in community, we also love experimenting and using new technologies. We ask if you do use ChatGPT or similar programs, please disclose what software and model version number you used to generate the response you used in your reply in order to promote transparency in the community.

ChatGPT answers in a tone that sounds like a tech support professional and while we support trying out this software, it is important to verify that they are solutions before posting otherwise it may mislead users into thinking that they are receiving official responses when they are not.

Per ChatGPT's website, the program does have limitations like:
- "May occasionally generate incorrect information"
- "May occasionally produce harmful instructions"
- "Limited knowledge of world and events after 2021"

**Comment by svr93** [comment]
@kingcc I can reproduce it using Chrome 106.0.5249.119. I found the problem code here:

<img width="1179" alt="image" src="https://github.com/community/community/assets/6628541/179211ef-81f9-47dd-80e8-f6d920fce6ab">

after disabling "display: revert" the behaviour become correct.

@the-officialjosh please do not spam here with unrelated AI-generated stuff

**Reply by kingcc** [reply]
Looks like they don't care about their user's browser environment 😂

**Comment by Lmb29860** [comment]
That is because the hacker above stole my get hub profile and has hit EVERY device I own!
